### PR TITLE
generic contextual menu for tooltip/responses

### PR DESCRIPTION
**Pull request by t3chguy**
Signed-off-by: Michael Telatynski <7t3chguy@gmail.com>

### CODE REVIEW

File: src/components/views/context_menus/GenericTextContextMenu.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
```
Comment (turt2live) [inline src/components/views/context_menus/GenericTextContextMenu.js]: inb4 licensing

Comment (t3chguy) [reply src/components/views/context_menus/GenericTextContextMenu.js]: heh, shows my copy pasta

Comment (t3chguy) [reply src/components/views/context_menus/GenericTextContextMenu.js]: done

File: src/components/views/context_menus/GenericTextContextMenu.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+'use strict';
+
+import React, {PropTypes} from 'react';
```
Comment (dbkr) [inline src/components/views/context_menus/GenericTextContextMenu.js]: We should movre to using `import PropTypes from 'prop-types'` in new code.

Comment (t3chguy) [reply src/components/views/context_menus/GenericTextContextMenu.js]: IIRC I tried but it wasn't installed

**Pull request merged**